packages/fxa-auth-server/config/index.ts

```diff
@@ -2046,6 +2046,44 @@ const convictConf = convict({
     },
   },
   cloudTasks: CloudTasksConvictConfigFactory(),
+  cloudScheduler: {
+    oidc: {
+      aud: {
+        default: '',
+        doc: 'The audience value of the id token payload.',
+        env: `AUTH_CLOUDSCHEDULER_OIDC_AUD`,
+        format: String,
+      },
+      serviceAccountEmail: {
+        default: '',
+        doc: 'The GCP service account email address.',
+        env: `AUTH_CLOUDSCHEDULER_OIDC_EMAIL`,
+        format: String,
+      },
+    },
+    deleteUnverifiedAccounts: {
+      sinceDays: {
+        default: 0,
+        doc: 'The time since which unverified accounts should be deleted.',
```

Comment:
Open to suggestions for variable name, this is how many days back from the current date to look.

Reply:
This was the default task limit used in the delete unverified script.